torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1374,6 +1374,20 @@ def sample_inputs_index_fill(op_info, device, dtype, requires_grad, **kwargs):
             samples.append(SampleInput(tensor, args=(d, idx, fill_val)))
             samples.append(SampleInput(tensor, args=(d, -idx - 1, fill_val)))
             samples.append(SampleInput(tensor, args=(d, idx_nonctg, fill_val)))
+
+    make_arg = partial(make_tensor, device=device, dtype=dtype, requires_grad=requires_grad)
+    index_tensor = partial(torch.tensor, device=device, dtype=torch.long)
+
+    def unique_idx(numel, max_idx):
+        indices = random.sample(range(max_idx), numel)
+        return index_tensor(indices)
```

Comment:
Non duplicate indices

cc: @mruberry

Reply:
Add a comment for this function

Good thing the tensors we're dealing with are small